kms-commands.go

```diff
@@ -107,8 +108,28 @@ func (adm *AdminClient) KMSStatus(ctx context.Context) (KMSStatus, error) {
 	return status, nil
 }
 
+// KMSStatusV2 returns status information about the KMS connected
+// to the MinIO server, if configured.
+func (adm *AdminClient) KMSStatusV2(ctx context.Context) (KMSStatus, error) {
```

Comment:
This can be changed to KMStatus() there is no reason to keep older API? since both return same values no client will break.

Reply:
thoughts @aead ^^?

Reply:
I believe it should be fine as long as [this PR](https://github.com/minio/minio/pull/15737) gets merged before this one

Reply:
That's not a problem @reivaj05 madmin-go will have a release tag only meant to be used with latest MinIO release, we can also make madmin-go move to `v2.0.0` so this way it indicates a large breakage.

Reply:
can you make the relevant changes? @reivaj05

Reply:
Sure, will do @harshavardhana